manifests/params.pp

```diff
@@ -204,15 +211,12 @@
         'WINEVENTLOG_FWD_ENABLE=1',
         'WINEVENTLOG_SET_ENABLE=1',
         'ENABLEADMON=1',
-        { 'INSTALLDIR' => $forwarder_homedir },
       ]
-      $enterprise_install_options = [
-        'LAUNCHSPLUNK=1',
-        'WINEVENTLOG_APP_ENABLE=1',
-        'WINEVENTLOG_SEC_ENABLE=1',
-        'WINEVENTLOG_SYS_ENABLE=1',
-        'WINEVENTLOG_FWD_ENABLE=1',
-        'WINEVENTLOG_SET_ENABLE=1',
+      $enterprise_install_options     = [
+        { 'INSTALLDIR' => $enterprise_homedir },
+        { 'SPLUNKD_PORT' => "${splunkd_port}" },
```

Comment:
If you need to disable a puppet-lint warning, you can use an end-of-line [control comment](http://puppet-lint.com/controlcomments/).
Do you need to do this to convert an Integer to a string?

Reply:
Instead of the control comment, I think you could also do...
```puppet
{ 'SPLUNKD_PORT' => String($splunkd_port) },
```

Reply:
oh cool, didn't know about the control comment. Thanks for the tip!
I'll use the String() function and push the update